### PR TITLE
muso: init at 2.0.0

### PR DESCRIPTION
**Pull request by ghost**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/muso/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  postInstall = ''
+    mkdir -p $out/share/muso
+    cp share/* $out/share/muso/
+  '';
+
+  cargoSha256 = "06jgk54r3f8gq6iylv5rgsawss3hc5kmvk02y4gl8iwfnw4xrvmg";
+
+  meta = with lib; {
+    description = "An automatic music sorter (based on ID3 tags).";
```
Comment (fabaff) [inline pkgs/applications/audio/muso/default.nix]:
```suggestion
    description = "An automatic music sorter (based on ID3 tags)";
```

see https://nixos.org/manual/nixpkgs/unstable/#sec-standard-meta-attributes

File: pkgs/applications/audio/muso/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    homepage = "https://github.com/quebin31/muso";
+    license = with licenses; [ gpl3Only ];
```
Comment (fabaff) [inline pkgs/applications/audio/muso/default.nix:36]: The license seems to be [GPLv3+](https://github.com/quebin31/muso/blob/main/cli/error.rs#L5-L8).

```suggestion
    license = with licenses; [ gpl3Plus ];
```

File: pkgs/applications/audio/muso/default.nix [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+with rustPlatform;
+
+buildRustPackage rec {
+  pname = "muso";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "quebin31";
+    repo = "muso";
```
Comment (fabaff) [inline pkgs/applications/audio/muso/default.nix:13]:
```suggestion
    repo = pname;
```

Not a blocker, more of a commonly done thing.

File: pkgs/applications/audio/muso/default.nix [hunk 4]
```diff
@@ -0,0 +1,39 @@
+{ lib, fetchFromGitHub, rustPlatform
+, pkgconfig, wrapGAppsHook
+}:
+
+with rustPlatform;
+
+buildRustPackage rec {
```
Comment (fabaff) [inline pkgs/applications/audio/muso/default.nix:7]:
```suggestion
rustPlatform.buildRustPackage rec {
```

**Review by fabaff — CHANGES_REQUESTED** [review]

File: pkgs/applications/audio/muso/default.nix (hunk 1 above)
Comment (bloomvdomino) [reply pkgs/applications/audio/muso/default.nix]: Already noticed ;)

Comment (fabaff) [reply pkgs/applications/audio/muso/default.nix]: My comment is already marked as outdated.

File: pkgs/applications/audio/muso/default.nix (hunk 2 above)
Comment (bloomvdomino) [reply pkgs/applications/audio/muso/default.nix]: Thanks, didn't noticed!

File: pkgs/applications/audio/muso/default.nix [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    rev = "68cc90869bcc0f202830a318fbfd6bb9bdb75a39";
+    sha256 = "1dnfslliss173igympl7h1zc0qz0g10kf96dwrcj6aglmvvw426p";
+  };
+
+  nativeBuildInputs = [ pkg-config wrapGAppsHook ];
+
+  preConfigure = ''
+    substituteInPlace lib/utils.rs \
+      --replace "/usr/share/muso" \
+                "$out/share/muso"
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/muso/default.nix:21]:
```suggestion
      --replace "/usr/share/muso" \
                "$out/share/muso"
```
```suggestion
      --replace "/usr/share/muso" "$out/share/muso"
```

**Pull request merged**